src/librustc_lint/unused.rs

```diff
@@ -392,7 +392,7 @@ impl UnusedParens {
         avoid_or: bool,
         avoid_mut: bool,
     ) {
-        use ast::{PatKind, BindingMode::ByValue, Mutability::Mutable};
+        use ast::{PatKind, BindingMode::ByValue, Mutability::Mut};
```

Comment:
I think we should just import `Mutability` here instead of the variant

Reply:
Done.